### DISCUSSION TITLE
Custom Badges for sensors

### DISCUSSION THREAD

**Discussion by serbanbiliuti** [dwainscheeren/dwains-lovelace-dashboard#791] [Dwains Dashboard V3 Alpha]
Hello,

Can anyone tell me a way to add badges in Dwains dashboard for custome sensors? 
For example I would like a badge that shows me the PV Instant generation in kW. 

I've been looking but didn't manage to find a specific file or section where to add some configuration. Also an example configuration that I can modify would help a lot.

Regards,
Serban

**Comment by dwainscheeren** [comment]
Hi,

I am closing the old Dwains Dashboard 3.x discussions as part of the DD3 cleanup.

DD3 is now in maintenance/community mode and is slowly being phased out. Active development is focused on Dwains Dashboard Next, the newest dashboard project:

https://github.com/dwainscheeren/dwains-dashboard-next

For chat, community help and discussion around DD3/DD Next, please join the Discord:

https://discord.gg/7yt64uX

If you still have a reproducible DD3 bug on the latest 3.9.x release, please open a new issue with current details. For new feature ideas, please try Dwains Dashboard Next.
